common/e2e_config/e2e_config.go

```diff
@@ -137,6 +137,7 @@ type ProductSpec struct {
 	LvmEnginePluginContainerName      string            `yaml:"lvmEnginePluginContainerName"`
 	LvmEnginePluginDriverName         string            `yaml:"lvmEnginePluginDriverName"`
 	UmbrellaOpenebsHelmChartName      string            `yaml:"umbrellaOpenebsHelmChartName"`
+	UseUmbrellaOpenEBSChart           bool              `yaml:"useUmbrellaOpenEBSChart" env:"e2e_openebs_chart" env-default:"false"`
```

Comment:
Can we have an environment variable name which reflects the field name and usage
so something like `e2e_use_umbrella_openebs_chart` ?

Reply:
Addressed